code/modules/spells/changeling/split.dm

```diff
@@ -0,0 +1,107 @@
+/spell/changeling/split
+	name = "Split (50)"
+	desc = "Split your body into two lifeforms."
+	abbreviation = "SP"
+	hud_state = "split"
+	charge_max = 300
+	cooldown_min = 30 SECONDS
+	still_recharging_msg = "<span class='warning'>We are not ready to do that!</span>"
+	chemcost = 50
+	required_dna = 1
+	max_genedamage = 0
+	horrorallowed = 0
+	var/datum/mind/owner = null // The mind of the user, to be used by the recruiter
+	var/datum/recruiter/recruiter = null
+	var/polling_ghosts = FALSE
+	
+/spell/changeling/split/Destroy()
+	owner = null
+	//qdel(recruiter)
+	recruiter = null
+	..()
+	
+/spell/changeling/split/cast(var/list/targets, var/mob/living/carbon/human/user)
+	owner = user.mind
+	var/datum/role/changeling/changeling = owner.GetRole(CHANGELING)
+	if (changeling.splitcount < 2) //<2 is two splits max
+		user.visible_message("[user] is preparing to generate a new form.")
+		Splitting()
+	else
+		user.visible_message("You are unable to split again.")
+	return
+
+/spell/changeling/split/proc/Splitting()
+	if(polling_ghosts)
+		return
+	polling_ghosts = TRUE
```

Comment:
???
alleged 2 splits max but this proc will only ever run once?
for what purpose